server/utils/placements/index.test.ts

```diff
@@ -0,0 +1,208 @@
+      expect(arrivalInformation(placement)).toEqual({
+        rows: [
+          {
+            key: { text: 'Expected arrival date' },
+            value: { text: DateFormats.isoDateToUIDate(placement.expectedArrivalDate) },
+          },
+          {
+            key: { text: 'Actual arrival date' },
+            value: { text: DateFormats.isoDateToUIDate(placement.actualArrivalDate) },
+          },
+          {
+            key: { text: 'Arrival time' },
+            value: { text: DateFormats.timeFromDate(DateFormats.isoToDateObj(placement.actualArrivalDate)) },
+          },
+          {
+            key: { html: '<span class="text-grey">Non arrival reason</span>' },
+            value: { html: '<span class="text-grey">-</span>' },
+          },
+          {
+            key: { html: '<span class="text-grey">Non arrival any other information</span>' },
+            value: { html: '<span class="text-grey">-</span>' },
+          },
+        ],
+      })
+    })
+
+    it('should return the departure information', () => {
+      expect(departureInformation(placement)).toEqual({
+        rows: [
+          {
+            key: { text: 'Expected departure date' },
+            value: { text: DateFormats.isoDateToUIDate(placement.expectedDepartureDate) },
+          },
+          {
+            key: { text: 'Actual departure date' },
+            value: { text: DateFormats.isoDateToUIDate(placement.actualDepartureDate) },
+          },
+          {
+            key: { text: 'Departure time' },
+            value: { text: DateFormats.timeFromDate(DateFormats.isoToDateObj(placement.actualDepartureDate)) },
+          },
+          { key: { text: 'Departure reason' }, value: { text: placement.departureReason?.name } },
+          {
+            key: { html: '<span class="text-grey">Breach or recall</span>' },
+            value: { html: '<span class="text-grey">-</span>' },
+          },
+          { key: { text: 'Move on' }, value: { text: placement.departureMoveOnCategory?.name } },
+          {
+            key: { html: '<span class="text-grey">More information</span>' },
+            value: { html: '<span class="text-grey">-</span>' },
+          },
+        ],
+      })
+    })
+  })
+  describe('otherBookings', () => {
+    it('should return a list of other bookings from a placement', () => {
+      const placementList = [
+        { id: 'id1', canonicalArrivalDate: '2024-09-10', canonicalDepartureDate: '2025-06-04' },
+        { id: 'id2', canonicalArrivalDate: '2024-09-20', canonicalDepartureDate: '2025-03-20' },
+      ]
+
+      const placement = cas1SpaceBookingFactory.build({ otherBookingsInPremisesForCrn: placementList })
+      expect(otherBookings(placement)).toEqual({
+        rows: [
+          {
+            key: { text: 'Other placement bookings at this premises' },
+            value: {
+              html: '<a href="id1">Placement 10 Sep 2024 to 04 Jun 2025</a><br/><a href="id2">Placement 20 Sep 2024 to 20 Mar 2025</a>',
```

Comment:
Should this `href` be the actual path?

Also it would be good to use an actual list for accessibility here.

Reply:
Actually, the id alone is a valid relative path, given that we are already on a placement detail page. It would probably be better to include a full path though.

Reply:
Oh, I see -- yeah, full path would be better for discoverability if anything (finding all places a certain path is used in code).